es_ES/faq.md

```diff
@@ -0,0 +1,625 @@
+ayuda a reducir los costos de desarrollo y mantenimiento de la aplicación.
+
+### Qué sucede cuando mi sistema operativo móvil se actualiza e introduce nuevos widgets?
+
+El equipo de Flutter observa la adopción y demanda de nuevos widgets para dispositivos móviles de iOS y Android, y tiene como objetivo trabajar con la comunidad para construir soporte para nuevos widgets. Este trabajo puede venir en la forma de las características del framework de nivel inferior, nuevos widgets compuestos, o nuevas implementaciones de widgets.
+
+La arquitectura en capas de Flutter está diseñada para soportar numerosas
+bibliotecas de widgets, y alentamos y apoyamos a la comunidad en
+construir y mantener bibliotecas de widgets.
+
+### Qué sucede cuando mi sistema operativo móvil se actualiza e introduce nuevas capacidades de plataforma?
+
+El sistema de interoperabilidad y plugin de Flutter está diseñado para permitir
+desarrolladores para acceder a las nuevas funciones y capacidades del sistema operativo móvil inmediatamente. Los desarrolladores no tienen que esperar al equipo de Flutter para exponer la nueva capacidad del sistema operativo móvil.
+
+### Qué sistemas operativos puedo usar para construir una aplicación de Flutter?
+
+Flutter admite el desarrollo en Linux, Mac y Windows.
+
+### En qué idioma está escrito Flutter?
+
+Observamos muchos lenguajes y tiempos de ejecución, y finalmente adoptamos a Dart para el framwork y los widgets. El framework de gráficos subyacente y la máquina virtual de Dart se implementan en C/C++.
+
+### Por qué Flutter eligió usar Dart?
+
+Flutter utilizó cuatro dimensiones principales para la evaluación, y consideró las necesidades de autores de framework, desarrolladores y usuarios finales. Encontramos algunos lenguajes encontrados algunos requisitos, pero Dart obtuvo una puntuación alta en todas nuestras dimensiones de evaluación y cumplió todos nuestros requisitos y criterios.
+
+Los tiempos de ejecución Dart y los compiladores admiten la combinación de dos características críticas para Flutter: un ciclo de desarrollo rápido basado en JIT que permite el cambio de forma y hot reload con estado en un idioma con tipado, más un compilador AOT que emite un código ARM eficiente para un arranque rápido y un rendimiento predecible de implementaciones de producción.
+
+Además, tenemos la oportunidad de trabajar estrechamente con la comunidad Dart,
+que está invirtiendo activamente recursos para mejorar Dart para usar en Flutter. por
+ejemplo, cuando adoptamos Dart, el lenguaje no tenía una AOT toolchain para producir binarios nativos, que es fundamental para lograr predecible, de alto rendimiento, pero ahora el lenguaje lo hace porque el equipo de Dart lo construyó para Flutter. Del mismo modo, la Dart VM se ha optimizado previamente para rendimiento pero el equipo ahora está optimizando la VM para la latencia, que es más

```

Comment:
which is instrumental in achieving predictable, high performance,
que es fundamental para lograr un alto rendimiento predecible